mkdocs.yml

```diff
@@ -5,565 +5,566 @@ repo_url: https://github.com/ezsystems/developer-documentation
-            - 'Limitations': 'guide/limitations.md'
-            - 'Limitation reference': 'guide/limitation_reference.md'
-            - 'Custom Policies': 'guide/custom_policies.md'
-        - User management:
-            - 'User management': 'guide/user_management/user_management.md'
-            - 'Customer groups': 'guide/user_management/customer_groups.md'
-            - 'Login and registration': 'guide/user_management/login_and_registration.md'
-            - 'OAuth authentication': 'guide/user_management/oauth.md'
-            - 'Add login through external service': 'guide/user_management/login_via_external_service.md'
-            - 'Token': 'guide/user_management/token.md'
-            - Customers:
-                - 'Customers': 'guide/customers/customers.md'
-                - 'Customer API': 'guide/customers/customer_api/customer_api.md'
-                - 'Customer profile data': 'guide/customers/customer_api/customer_profile_data.md'
-                - 'Customer data configuration': 'guide/customers/customer_api/configuration_for_customer_data.md'
+            - Configuration: guide/configuration/configuration.md
+            - Dynamic configuration: guide/configuration/config_dynamic.md
+            - Repository configuration: guide/configuration/config_repository.md
+        - Extend Back Office:
+            - Back Office: extending/extending_back_office.md
+            - Configuration: extending/config_back_office.md
+            - Content Tree: extending/content_tree.md
+            - Back Office elements:
+                - Reusable components: extending/reusable_back_office_templates.md
+                - Add drop-downs: extending/creating_custom_dropdowns.md
+                - Custom icons: extending/custom_icons.md
+                - Add drag and drop: extending/drag_and_drop.md
+                - Custom components: extending/custom_components.md
+                - Formatting date and time: extending/extending_date_and_time.md
+                - Extending thumbnails: extending/extending_thumbnails.md
+                - Importing assets from a bundle: extending/import_assets_from_bundle.md
+            - Back Office tabs:
+                - Back Office tabs:  extending/tabs/back_office_tabs.md
+                - Create dashboard tab: extending/tabs/add_dashboard_tab.md
+            - Back Office menus:
+                - Back Office menus: extending/menus/back_office_menus.md
+                - Add menu item: extending/menus/add_menu_item.md
+            - Add user setting: extending/add_user_setting.md
+            - Customize calendar: extending/customize_calendar.md
+            - Browser:
+                - Browser: extending/extending_udw.md
+                - Add browser tab: extending/add_udw_tab.md
+            - Multi-file upload: extending/extending_multifile_upload.md
+            - Sub-items list: extending/extending_subitems_list.md
+    - Content rendering:
```

Comment:
Two problematic areas are Content Rendering and Content Management, as files from them in many cases can freely flow between both categories, it would be good to establish what are criteria for each category and reassess.

Reply:
Idea: 
Content Rendering -> Rendering content (anything that's concerned with displaying content on the front)
Content Management -> Content (anything that's concerned with editing / managing content in the Back Office or in code). I'm still not sure about the place of Admin Panel (https://doc.ibexa.co/en/master/guide/admin_panel/) in this structure, maybe it just needs rewriting (again).

A lot other solutions use "Templating" in place of "Content rendering", to be considered.